java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -613,4 +614,81 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Long> zremrangebylex(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.<br>
+     * To get the elements with their scores, see {@link #zunionWithScores}.
+     *
```

Comment:
I think you have to add a cross slot notice for cluster client as it was requested in https://github.com/aws/glide-for-redis/pull/1248#discussion_r1567026628
(for all functions)